### PR TITLE
#Issue SB-5093 Feat:  language support on the portal

### PR DESCRIPTION
**Pull request by ArchanaKumari01**

### CODE REVIEW

**Comment by harishgilimi** [comment]
@anoophm  check this implementation design and update @ArchanaKumari01

File: src/app/client/src/app/modules/core/components/data-driven-filter/data-driven-filter.component.ts [hunk 1]
```diff
@@ -57,6 +58,8 @@ export class DataDrivenFilterComponent implements OnInit, OnChanges {
   }
 
   ngOnInit() {
+    // subscribe this.resourceService.languageSelected$
+    // check if formFieldProperties length is greater than 0 and check formInput data have value then call a function to change values
```
Comment (anoophm) [inline src/app/client/src/app/modules/core/components/data-driven-filter/data-driven-filter.component.ts]: any one has value then do the conversation

File: src/app/client/src/app/modules/shared/components/page-section/page-section.component.ts [hunk 2]
```diff
@@ -122,6 +122,13 @@ export class PageSectionComponent implements OnInit {
     this.playEvent.emit(event);
   }
   ngOnInit() {
+    this.resourceService.languageSelected$
```
Comment (anoophm) [inline src/app/client/src/app/modules/shared/components/page-section/page-section.component.ts]: unsubscribe from all laug subscription

File: src/app/client/src/app/modules/shared/services/util/util.service.ts [hunk 3]
```diff
@@ -88,4 +88,73 @@ export class UtilService {
...
+        this.convert(value, lang);
+        _.forEach(value.children, (children) => {
+          this.convert(children, lang);
+        });
+      } else if (value.translations) {
+        this.convert(value, lang);
+      }
+    });
+    const translate = data;
+    return translate;
```
Comment (anoophm) [inline src/app/client/src/app/modules/shared/services/util/util.service.ts]: return data

**Pull request merged**